### PR TITLE
Update/device app action

### PR DESCRIPTION
**Pull request by shaunmulligan**
updated device and app level actions in the management section

### CODE REVIEW

File: pages/management/devices.md [hunk 1]
```diff
@@ -46,6 +49,12 @@ To see what your device is serving on port 80, just click on the URL. If your ap
 
 __Note:__ Currently websockets will not work over the device URLs, this should be resolved, keep an eye on the [github issue](https://github.com/resin-io/node-tunnel/issues/2) for an update of this feature.
 
+### Update Locking
+
+In many uses cases devices are performing sensitive or critical functionality and are not able to pause to receive an update or restart the container. For this reason we add the [update.lock functionality](/pages/runtime/supervisor-api.md#update-locking) in the resin supervisor agent. This allows your application to pick and choose when and where it would like to allow updates to happen.
+
+Added to this functionality we provided a convenient to override the lock on the device and essentially force an update. This is a precautionary measure for those times when your application crashes and hasn't released the update lock. This gives you a nice safety net to ensure you can always push new updates.
```
Comment (dfunckt) [inline pages/management/devices.md]: "we provided a convenient to override the lock" -> "we provided a convenient button (?) to override the lock"

File: pages/management/devices.md [hunk 2]
```diff
@@ -46,6 +49,12 @@ To see what your device is serving on port 80, just click on the URL. If your ap
 
 __Note:__ Currently websockets will not work over the device URLs, this should be resolved, keep an eye on the [github issue](https://github.com/resin-io/node-tunnel/issues/2) for an update of this feature.
 
+### Update Locking
+
+In many uses cases devices are performing sensitive or critical functionality and are not able to pause to receive an update or restart the container. For this reason we add the [update.lock functionality](/pages/runtime/supervisor-api.md#update-locking) in the resin supervisor agent. This allows your application to pick and choose when and where it would like to allow updates to happen.
```
Comment (dfunckt) [inline pages/management/devices.md]: "For this reason we add the" -> "For this reason we add**ed** the"

**Comment by shaunmulligan** [comment]
@dfunckt thanks and updated

File: pages/management/devices.md [hunk 3]
```diff
@@ -78,6 +87,18 @@ On all resin.io devices `/data` is a persistent data volume. This is useful for
 
 It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
```
Comment (dfunckt) [inline pages/management/devices.md]: Maybe also add the "version available" note here too, as you did with the app action?

``__Warning:__ This action is only supported on devices with an Agent version >= 1.1.0``

**Pull request merged**